.github/workflows/flake8.yml

```diff
@@ -0,0 +1,16 @@
+name: Flake8
+on: pull_request
+job:
+flake8:
```

Comment:
please use indentations

Reply:
It should've had indentations already. I reuploaded the file, is it showing up right?

Reply:
yes not it's looking good 
Pay attention - this is only a review on your code but I can't validate that it's actually working
Are you sure it's working on your ENV?

Reply:
It looks like it's working on my end